src/metrics/metrics.go

```diff
@@ -50,8 +61,21 @@ func PopulateInstanceMetrics(instanceEntity *integration.Entity, connection *con
 	}
 
 	populateWaitTimeMetrics(instanceEntity, connection)
-	if arguments.CustomMetricsQuery != "" {
-		populateCustomMetrics(instanceEntity, connection, arguments.CustomMetricsQuery)
+	if len(arguments.CustomMetricsQuery) > 0 {
+		populateCustomMetrics(instanceEntity, connection, []CustomQuery{{Query: arguments.CustomMetricsQuery}})
+	} else if len(arguments.CustomMetricsConfig) > 0 {
+		// load YAML config file
+		b, err := ioutil.ReadFile(arguments.CustomMetricsConfig)
+		if err != nil {
+			log.Fatal(err)
```

Comment:
This should log at error level, then return from the function. We generally avoid panicking, which is what the SDK's `log.Fatal` does

Reply:
updated code